docs/pact-reference.md

```diff
@@ -821,6 +821,66 @@ and [pact-state](#pact-state) repl functions to simulate pact executions.
 
 It is not possible yet (as of Pact 2.3.0) to simulate pact execution in the pact server API.
 
+Dependency Management
+---
+Pact supports a number of features to manage a module's dependencies on other Pact modules.
+
+### Module Hashes
+Once loaded, a Pact module is associated with a hash computed from the module's source code text.
+This module hash uniquely identifies the version of the module.
+Module hashes can be examined with [describe-module](#describe-module):
+
+```
+pact> (at "hash" (describe-module 'accounts))
+"9d6f4d3acb2fd528206330d09a8926da6abdd9ac5e8c4b24cc35955203f234688c25f9545ead56f783c5269fe4be6a62aa89162caf811142572ac172dc2adb91"
+```
+
+### Pinning module versions with `use`
+The [use](#use) special form allows
+a module hash to be specified, in order to pin the dependency version. When
+used within a module declaration, it introduces the dependency
+hash value into the module's hash.
+This allows a "dependency-only" upgrade to push the upgrade to the module version.
+
+### Inlined Dependencies: "No Leftpad"
```

Comment:
I'm not sure we can expect the audience to know the leftpad debacle

Reply:
I go back and forth on this, leftpad comes up often in the discussions of the Parity bug online. But perhaps it's not the right tone for a reference?